readme.md

```diff
@@ -157,14 +157,44 @@ Returns a `Stream` instead of a `Promise`. This is equivalent to calling `got.st
 
 Type: `string` `Buffer` `stream.Readable` [`form-data` instance](https://github.com/form-data/form-data)
 
-**Note:** If you provide this option, `got.stream()` will be read-only.
+**Note:** The `body` option cannot be used with the `json` or `form` option.
 
-The body that will be sent with a `POST` request.
+**Note:** If you provide this option, `got.stream()` will be read-only.
 
 If present in `options` and `options.method` is not set, `options.method` will be set to `POST`.
 
 The `content-length` header will be automatically set if `body` is a `string` / `Buffer` / `fs.createReadStream` instance / [`form-data` instance](https://github.com/form-data/form-data), and `content-length` and `transfer-encoding` are not manually set in `options.headers`.
 
+###### json
+
+Type: `Object` `Array` `number` `string` `boolean` `null`
+
+**Note:** If you provide this option, `got.stream()` will be read-only.
+
+JSON body. The `Content-Type` header will be set to `application/json` if it's not defined.
+
+###### responseType
+
+Type: `string`<br>
+Default: `text`
+
+**Note:** When using streams, this option is ignored.
+
+Parsing method used to retrieve the body from the response. Can be `text`, `json` or `buffer`. The promise has `.json()` and `.buffer()` functions which set this option automatically.
```

Comment:
For completeness, it should have `.text()` too.

Reply:
it has. updating the readme now :)